### ISSUE TITLE
[Compatibility] RHS v0.4.1.1 and ACE3.  Adjusting mil causes vanilla zeroing to also increase

### ISSUE THREAD

**Issue by TheMagnetar** [acemod/ACE3#4162]
**Arma 3 Version:** `1.62` (stable)
**CBA Version:** `3.0.0` (stable)
**ACE3 Version:** `3.6.2` (stable)

**Mods:**
- `@CBA_A3`
- `@ace3`
- `@rhs_usaf v0.4.1.1

**Description:**
- RHS scopes do not work with advanced ballistics. When trying to increase the mils according to the atragMX both mils and vanilla range increase. This does not allow then to use ace3 system since vanilla zeroing always takes precedence.

* 0 mils -> vanilla zeroing 200m
* 0.1 mils -> vanilla zeroing 300m
* 0.2 mils -> vanilla zeroing 400m

This can be reproduced with M85 41A SSDS, MK 4ER/3.5 10x M3 and MK 4ER/T6.5-30xM5. It does not happen with LRPS scope.

[Imgur](http://i.imgur.com/T1UaOr7.jpg)
[Imgur](http://i.imgur.com/JqCmW7G.jpg)
[Imgur](http://i.imgur.com/Ud31SGb.jpg)

**Steps to reproduce:**
- Place optional rhs compat pbos in the addons folder.
- Equip a sniper rifle like 2010 ESR.
- Equip with a RHS scope: M85 41A SSDS, MK 4ER/3.5 10x M3 or MK 4ER/T6.5-30xM5
- Adjust mils with avpag
- Vanilla zeroing will also increase.
- Shoot the target according to the mils given by atragmx.
- Target will be missed since vanilla zeroing takes preference.

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer / Singleplayer /

**Placed Modules:**
- advanced ballistics.

**RPT log file:**
Not relevant

**Comment by esteldunedain** [comment]
Please provide RPT logs as per instructions. You most likely didn't load the RHS compatibility pbos from the optionals folder.

**Comment by PabstMirror** [comment]
I saw this as well, no need for rpt.
We change vanilla scopes to do 
```
                    discreteDistance[] = {100};
                    discreteDistanceInitIndex = 0;
```
so you can't vanilla zero them.
I think we need to do the same for RHS

**Comment by esteldunedain** [comment]
>I think we need to do the same for RHS

I thought we did that already :(

**Comment by J3aLeiNe** [comment]
The Premier scopes (M85) are not included in the compat_RHS 3.6.2 and you are right @PabstMirror.
I can fix it in my PR if you want, it's easy to do.

**Comment by PabstMirror** [comment]
Fixed by #4028
